records_mover/records/records_format.py

```diff
@@ -43,6 +48,11 @@ def __repr__(self) -> str:
     def generate_filename(self, basename: str) -> str:
         return f"{basename}.parquet"
 
+    def __eq__(self, other: object) -> bool:
+        if isinstance(other, ParquetRecordsFormat):
+            return True
+        return False
+
```

Comment:
The above `__eq__()` methods are primarily convenience things; they make the unit tests nicer when comparing.  We'll need to make them more complex once these records formats have hints associated with them (e.g., compression type).